lib/spack/spack/platforms/cray.py

```diff
@@ -4,30 +4,30 @@
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
 import os
+import os.path
 import re
+import platform
+import llnl.util.cpu as cpu
 import llnl.util.tty as tty
 from spack.paths import build_env_path
 from spack.util.executable import Executable
 from spack.architecture import Platform, Target, NoPlatformError
 from spack.operating_systems.cray_frontend import CrayFrontend
-from spack.operating_systems.cnl import Cnl
+from spack.operating_systems.cray_backend import CrayBackend
 from spack.util.module_cmd import module
 
 
-def _get_modules_in_modulecmd_output(output):
-    '''Return list of valid modules parsed from modulecmd output string.'''
-    return [i for i in output.splitlines()
-            if len(i.split()) == 1]
+_craype_name_to_target_name = {
+    'x86_cascadelake': 'cascadelake',
+    'x86_naples': 'zen',
+    'x86_rome': 'zen',  # Cheating because we have the wrong modules on rzcrayz
```

Comment:
What do we do about this one?  Can we fix the modules on `rzcrayz`?  I don't want this one merged to `develop`.